cli/tests/integration_tests.rs

```diff
@@ -5297,16 +5297,13 @@ fn web_platform_tests() {
   let config: std::collections::HashMap<String, Vec<WptConfig>> =
     deno_core::serde_json::from_value(jsonc_to_serde(jsonc)).unwrap();
 
-  // Observation: `python3 wpt serve` hangs with the python3 from homebrew
-  // but works okay with /usr/bin/python, which is python 2.7.10. Observed
-  // with homebrew python 3.8.5, 3.8.7 and 3.9.1.
   let python = match true {
     _ if cfg!(target_os = "windows") => "python.exe",
-    _ if cfg!(target_os = "macos") => "python",
     _ => "python3",
   };
 
   eprintln!("If the wpt server fails or gets stuck, please set up your /etc/hosts file like specified in //docs/contributing/building_from_source.md.");
+  eprintln!("On windows python.exe needs to available in your path, and needs to point to python 3. On macOS and Linux, python3 needs to be available in your path.");
```

Comment:
Maybe we should check this here in Rust? Like start `python.exe --version` and see that it has the correct output?

Reply:
Would it be ok if I do this when I move the WPT runner to TS?

Reply:
Not convinced about moving WPT runner to TS, but sure